docs/spec/staking/end_block.md

```diff
@@ -16,3 +16,34 @@ EndBlock() ValidatorSetChanges
     ClearTendermintUpdates()
     return vsc
 ```
+
+
+## CompleteUnbonding
+
+Complete the unbonding and transfer the coins to the delegate. Perform any
```

Comment:
How about "realize" any slashing, "perform" is a bit confusing since the slashing already happened. Let's explain how the slashing already occurred - namely, that the unbonding delegation struct will be updated at that point to reflect the reduced later withdrawal amount.